box-ui/src/components/ts/Student/StudentMeeting.tsx

```diff
@@ -72,9 +72,11 @@ const StudentMeeting: FunctionComponent = () => {
     }, [img_original]);
 
     useEffect(() => {
-        const address = process.env.REACT_APP_WS_ADDRESS;
-        if (address) {
-            const ws1 = new WebSocket(address);
+        const address_ws = process.env.REACT_APP_WS_ADDRESS;
```

Comment:
You should use camelCase as it is the convention for variables in the repo. (addressWs may be confusing so I recommend addressWS or addressWebsocket)
```suggestion
        const addressWebsocket = process.env.REACT_APP_WS_ADDRESS;
```